torch/_logging/_internal.py

```diff
@@ -120,6 +120,8 @@ def clear(self):
 
 # sample usage: torch._logging.set_logs(**torch._logging.DEFAULT_LOGGING)
 DEFAULT_LOGGING = {
+    "graph_code": True,
+    "aot": logging.INFO,
```

Comment:
Actually don't we want dynamo logging.INFO too?

Reply:
Added